systemtest/src/test/java/io/strimzi/systemtest/metrics/StrimziMetricsReporterST.java

```diff
@@ -0,0 +1,133 @@
+import io.strimzi.systemtest.TestConstants;
+import io.strimzi.systemtest.annotations.ParallelTest;
+import io.strimzi.systemtest.docs.TestDocsLabels;
+import io.strimzi.systemtest.performance.gather.collectors.BaseMetricsCollector;
+import io.strimzi.systemtest.resources.ResourceManager;
+import io.strimzi.systemtest.resources.crd.KafkaNodePoolResource;
+import io.strimzi.systemtest.templates.crd.KafkaNodePoolTemplates;
+import io.strimzi.systemtest.templates.crd.KafkaTemplates;
+import io.strimzi.systemtest.templates.crd.KafkaTopicTemplates;
+import io.strimzi.systemtest.templates.specific.ScraperTemplates;
+import io.strimzi.systemtest.utils.kafkaUtils.KafkaTopicUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.LockSupport;
+
+import static io.strimzi.systemtest.TestTags.ACCEPTANCE;
+import static io.strimzi.systemtest.TestTags.CRUISE_CONTROL;
+import static io.strimzi.systemtest.TestTags.METRICS;
+import static io.strimzi.systemtest.TestTags.REGRESSION;
+import static io.strimzi.systemtest.TestTags.SANITY;
+import static io.strimzi.systemtest.utils.specific.MetricsUtils.assertMetricCountHigherThan;
+import static io.strimzi.systemtest.utils.specific.MetricsUtils.assertMetricValue;
+import static io.strimzi.systemtest.utils.specific.MetricsUtils.assertMetricValueCount;
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
+
+@Tag(SANITY)
+@Tag(REGRESSION)
+@Tag(METRICS)
+@Tag(CRUISE_CONTROL)
+@SuiteDoc(
+    description = @Desc("This test suite is designed for testing metrics exposed by the Strimzi Metrics Reporter."),
+    beforeTestSteps = {
+        @Step(value = "Create namespace {@namespace}.", expected = "Namespace {@namespace} is created."),
+        @Step(value = "Deploy Cluster Operator.", expected = "Cluster Operator is deployed."),
+        @Step(value = "Deploy Kafka {@clusterName} with metrics.", expected = "Kafka @{clusterName} is deployed."),
+        @Step(value = "Deploy scraper Pod in namespace {@namespace} for collecting metrics from Strimzi pods.", expected = "Scraper Pods is deployed."),
+        @Step(value = "Create KafkaTopic resource.", expected = "KafkaTopic resource is Ready."),
+        @Step(value = "Create collector for Kafka.", expected = "Metrics collected in collectors structs.")
+    },
+    afterTestSteps = {
+        @Step(value = "Common cleaning of all resources created by this test class.", expected = "All resources deleted.")
+    },
+    labels = {
+        @Label(value = TestDocsLabels.METRICS),
+    }
+)
+public class StrimziMetricsReporterST extends AbstractST {
+    private static final Logger LOGGER = LogManager.getLogger(StrimziMetricsReporterST.class);
+
+    private final String namespace = Environment.TEST_SUITE_NAMESPACE;
+    private final String clusterName = "my-cluster";
```

Comment:
Instead of using this you can use the `TestStorage` as it is used in all other tests.